doc/source/whatsnew/v0.24.0.rst

```diff
@@ -643,7 +643,7 @@ Parsing datetime strings with different UTC offsets will now create an Index of
 Passing ``utc=True`` will mimic the previous behavior but will correctly indicate
 that the dates have been converted to UTC
 
-.. ipython:: python
+.. code-block:: ipython
 
     pd.to_datetime(["2015-11-18 15:30:00+05:30",
                     "2015-11-18 16:30:00+06:30"], utc=True)
```

Comment:
if you use `code-block:: ipython`, you'll have to explicitly show input and output

Reply:
Thank you, I corrected my mistake and updated the pr.